redis.js

```diff
@@ -0,0 +1,16 @@
+const redis = require("redis");
+const redisUrl = "redis://127.0.0.1:6379";
```

Comment:
We need to be able to set a custom value to `redisUrl` without changing the code. An environment variable will be useful here.
```suggestion
const redisUrl = process.env.REDIS_URL || "redis://127.0.0.1:6379";
```